output/alchemical_analysis.py

```diff
@@ -129,8 +129,18 @@ def output(self,  estimators, args):
         :return:
         """
         t = args.temperature
+
+        if args.unit == 'kT':
+            conversion = 1.0
+            args.unit = 'kT'
+        elif args.unit == 'kJ' or args.unit == 'kJ/mol':
+            conversion = 1.0 * t * self.k_b
+            args.unit = 'kJ/mol'
+        elif args.unit == 'kcal' or args.unit == 'kcal/mol':
+            conversion = 0.239006 * t * self.k_b
+            args.unit = 'kcal/mol'
+
```

Comment:
duplication of conversion code — can this be put into a separate, reusable function ?

(There's some discussion on units in alchemlyb https://github.com/alchemistry/alchemlyb/issues/125 but it will be a while until that is ready so in the meantime, flanel should certainly provide a way for unit conversion.)

Reply:
Putting this code and its duplicates in a separate function is definitely the best way. The linked discussion is very interesting though.

Reply:
By all means, contribute to the discussion! Contributors are very welcome.